core_backend/app/llm_call/check_output.py

```diff
@@ -105,8 +109,18 @@ async def _check_align_score(
     }
 
     if align_score.score < float(ALIGN_SCORE_THRESHOLD):
```

Comment:
I am having trouble coming up with a test case for this..  RAG needs to succeed, but alignment needs to fail!

Reply:
What if we just call `_check_align_score` with a made up `UserQueryResponse`?
It more a unit test than an end to end test.